core/App/screens/ProofRequest.tsx

```diff
@@ -256,7 +256,7 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
             }}
           >
             {loading ? <RecordLoading /> : null}
-            {proofConnectionLabel && <ConnectionAlert connectionID={proofConnectionLabel} />}
+            {!!proofConnectionLabel && <ConnectionAlert connectionID={proofConnectionLabel} />}
```

Comment:
I think the convention we tend to stick with is to use a ternary and return null if false. So 
```jsx
proofConnectionLabel ? (<... />): null
```

Reply:
change completed